src/client/app/app.js

```diff
@@ -33,7 +33,7 @@ angular.module('GitSpy', [
     templateUrl: '/app/dashboard/dashboard.html',
     controller: 'DashboardController'
   });
-  $routeProvider.when('/:orgName/:repoName/setup', {
+  $routeProvider.when('/:dashboardId/setup', {
```

Comment:
We still want the client side route to be '/:orgName/:repoName/setup', so we shouldn't alter this